### PR TITLE
21 users can find related links at the bottom of the site

### LINKED ISSUE #21: Users can  find related links at the bottom of the site
As a user, when I browse the website, I would like to find relevant links to the company, products and support at the bottom of the page so that I can learn more about the content and services of the website.

Related links:
The following links should be displayed in the footer section:
Company Related Links
Product related links
Support Links
Privacy Policy
Terms of Use
Each link should navigate to the appropriate page or section.

### PR DESCRIPTION
**Pull request by sguming**
image
[acf-export-2024-06-12.json](https://github.com/user-attachments/files/15804347/acf-export-2024-06-12.json)
这边title 使用空格 分辨几个title，link使用！换行#换列， 使用js来加入动作，代码在mainjs里面，具体function会等基本做完之后挨个实现，

### CODE REVIEW

**Comment by sguming** [comment]
之后也会做成function加入footer，但是由于上一个branch没弄好，function没法render所以暂时搁置在这去做下一个了

File: html/wp-content/themes/ailurotech/LInk.php
Comment (RuntaoZhuge) [inline html/wp-content/themes/ailurotech/LInk.php:1]: The Name is wrong

File: html/wp-content/themes/ailurotech/LInk.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * Template Name: Link Page
+ */
+
+use Timber\Timber;
+
+$context = Timber::context();
+
+// Get the current post
+$string = "about us!services!careers!partnership#fintech!software#Docs!Helpcenter";
+$title="Company Products Support";
```
Comment (RuntaoZhuge) [inline html/wp-content/themes/ailurotech/LInk.php]: What are these hardcoded values for?

File: html/wp-content/themes/ailurotech/src/scss/_link.scss
Comment (RuntaoZhuge) [inline html/wp-content/themes/ailurotech/src/scss/_link.scss:1]: This is an empty file

File: html/wp-content/themes/ailurotech/views/base.twig [hunk 2]
```diff
@@ -18,6 +18,6 @@
 
       {% endblock %}
     </main>
-    {% include 'partial/footer.twig' %}
+    {% include 'partial/link.twig' %}
```
Comment (RuntaoZhuge) [inline html/wp-content/themes/ailurotech/views/base.twig]: Don't need to remove footer

File: html/wp-content/themes/ailurotech/views/partial/link.twig
Comment (RuntaoZhuge) [inline html/wp-content/themes/ailurotech/views/partial/link.twig:1]: This name is too generic, needs more detailed naming

File: html/wp-content/themes/ailurotech/views/partial/link.twig [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        display: flex;
+        width:700px;
+        height:1000px;
+    }
+    .column {
+        flex: 1;
+        padding: 10px;
+    }
+    .title {
+        /* 标题的样式 */
```
Comment (RuntaoZhuge) [inline html/wp-content/themes/ailurotech/views/partial/link.twig]: What is this empty css

File: html/wp-content/themes/ailurotech/views/partial/link.twig [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    }
+    .link {
+        color: #6A5ACD; /* 蓝紫色 */
+        margin-top: 5px; /* 设置上边距 */
+    }
+</style>
```
Comment (RuntaoZhuge) [inline html/wp-content/themes/ailurotech/views/partial/link.twig]: We don't use inline css or css in twig. This is most likely to fail

File: html/wp-content/themes/ailurotech/views/partial/link.twig [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+<div class="container">
+    {% for column in link %}
+        <div class="column">
+            <h2 class="title">{{ titles[loop.index0] }}</h2> <!-- 为标题设置 class="title" -->
```
Comment (RuntaoZhuge) [inline html/wp-content/themes/ailurotech/views/partial/link.twig:8]: This class is very generic

File: html/wp-content/themes/ailurotech/views/partial/link.twig [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+            {% for item in column %}
+                <span class="link">{{ item }}</span><br> <!-- 为链接设置 class="link" -->
```
Comment (RuntaoZhuge) [inline html/wp-content/themes/ailurotech/views/partial/link.twig]: This class is very generic

**Review by RuntaoZhuge — CHANGES_REQUESTED** [review]
There are some changes to be made and don't just straight copy and paste what chatgpt dump there, you need to read the code before you submit a pull request

**Pull request closed without merging**